docs/data/material/getting-started/templates/dashboard/components/MainGrid.js

```diff
@@ -44,18 +44,16 @@ export default function MainGrid() {
   return (
     <React.Fragment>
       {/* cards */}
-
       <Grid container spacing={2} columns={12}>
         {data.map((card, index) => (
-          <Grid xs={6} sm={3} md={2} lg={2.25}>
-            <StatCard key={index} {...card} />
+          <Grid key={index} xs={6} sm={3} md={2} lg={2.25}>
```

Comment:
What do you think about this diff instead?

```suggestion
          <Grid key={card.title} xs={6} sm={3} md={2} lg={2.25}>
```

Reply:
I could see the same title with different time ranges: https://deploy-preview-42594--material-ui.netlify.app/material-ui/getting-started/templates/dashboard/. I don't think it matters here that it uses the array index. It will be more flexible for people.

Reply:
Ah okay, was just commenting based on `data` in this file.